### ISSUE TITLE
USB disks not detected in disk/detect but USB tests Passed Correctly

### ISSUE THREAD

**Issue by mreed8855** [canonical/checkbox#2801]
### Bug Description

The USB disk were not detected by disk/detect but they were detected by the USB tests and they passed. 
Name: /dev/sda
	Model: 	LOGICAL VOLUME
	Size:  	745.2G
Name: /dev/sdb
	Model: 	PRAID CP700i
	Size:  	744.7G
Name: /dev/sdc
	Model: 	PRAID EP740i
	Size:  	744.7G

sdd and sde are not in disk detect but they are in in usb/storage-server and usb3/storage-server

--------------------
Removable devices currently mounted:
None
Removable devices currently not mounted:
/dev/sdd
/dev/sde
--------------------
Found the following mounted usb partitions:
    /dev/sdd : /tmp/tmp83rp3ay6 : 5000000000 bits/s
    /dev/sde : /tmp/tmp9x4f56ol : 5000000000 bits/s
--------------------
/dev/sdd (Total Data Size / iteration: 255.9662 MB):
	[Iteration 0] Average Speed: 4.9142
	Summary:
		Total Data Attempted: 255.9662 MB
		Total Time to write: 52.0867 secs
		Average Write Time: 52.0867 secs
		Average Write Speed: 4.9142 MB/s
/dev/sde (Total Data Size / iteration: 255.9662 MB):
	[Iteration 0] Average Speed: 4.3112
	Summary:
		Total Data Attempted: 255.9662 MB
		Total Time to write: 59.3720 secs
		Average Write Time: 59.3720 secs
		Average Write Speed: 4.3112 MB/s

### Cert-blocker Test Case

- [ ] cert-blocker

### To Reproduce

1. Install canonical-certification-server
2. Run certify-ubuntu

### Expected Result

The expected result is that the usb drives are listed in the output of  disk-detect

### Actual Result

The USB drives sdd and sde are missing from the listing
 
Name: /dev/sda
	Model: 	LOGICAL VOLUME
	Size:  	745.2G
Name: /dev/sdb
	Model: 	PRAID CP700i
	Size:  	744.7G
Name: /dev/sdc
	Model: 	PRAID EP740i
	Size:  	744.7G

### Environment

Ubuntu 24.04.4 (6.8.0-136-generic)
checkbox-provider-certification-server	7.3.0~dev42~ubuntu24.04.1

### Relevant log output

```shell

```
